components/suivi-form/livrables/stockage-form/ftp-form.js

```diff
@@ -0,0 +1,143 @@
+import {useState} from 'react'
+import PropTypes from 'prop-types'
+
+import TextInput from '@/components/text-input.js'
+import NumberInput from '@/components/number-input.js'
+import Button from '@/components/button.js'
+
+const FtpForm = ({initialValues, onSubmit, onCancel}) => {
+  const [values, setValues] = useState(initialValues)
+  const [checkedStatus, setCheckedStatus] = useState(false)
```

Comment:
Pas besoin d'avoir une state dédier puisque la valeur est `values.secure`. On peut adapter l'initialisation pour s'assurer de lui donner une valeur.
```suggestion
  const [values, setValues] = useState({...initialValues, secure: initialValues.secure || false})
```